core/templates/dev/head/pages/collection_editor/editor_tab/CollectionLinearizerService.js

```diff
@@ -205,12 +208,15 @@ oppia.factory('CollectionLinearizerService', [
           return false;
         }
 
+        var linearNodeList = _getCollectionNodesInPlayableOrder(collection);
+        var nodeIndex = findNodeIndex(linearNodeList, explorationId);
+        var collectionNode = linearNodeList[nodeIndex];
+        var acquiredSkill =
+          collectionNode.getAcquiredSkillList().getSkills()[0];
```

Comment:
I'm a bit iffy about using index 0; is there a better way for us to find out the skill ID associated with the exploration ID? Maybe using the exploration title to look up the corresponding skill ID? I'm not sure whether we can make the guarantee that acquired skills are going to remain ordered, or if we should.